sel4-ftrace-merge/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# set minimum cmake verision
+cmake_minimum_required(VERSION 3.0.0)
+
+# set project name
+project(ftrace-merge)
+
+# add executable
+add_executable (ftrace-merge ftrace-merge.c)
+
+# set compiler options
+set(FLAGS_WAR_AS_ERR -Wall -Wpedantic -Werror)
+# set(FLAGS_MEM_DBG -fsanitize=address -fsanitize=leak -fsanitize=bounds)
```

Comment:
same comment apply here as with extract. In fact you could define these flags in common CMakeLists.txt (i.e. one dir above.